src/WebJobs.Script/Binding/DocumentDBScriptBindingProvider.cs

```diff
@@ -115,6 +115,13 @@ public override Collection<Attribute> GetAttributes()
                     attributeTrigger.LeaseConnectionStringSetting = Context.GetMetadataValue<string>("leaseConnectionStringSetting");
                     attributeTrigger.CreateLeaseCollectionIfNotExists = Context.GetMetadataValue<bool>("createLeaseCollectionIfNotExists", false);
                     attributeTrigger.LeasesCollectionThroughput = Context.GetMetadataValue<int>("leaseCollectionThroughput", 0);
+                    attributeTrigger.LeaseCollectionPrefix = Context.GetMetadataValue<string>("leaseCollectionPrefix");
+                    attributeTrigger.FeedPollDelay = Context.GetMetadataValue<int>("feedPollDelay", 0);
+                    attributeTrigger.LeaseAcquireInterval = Context.GetMetadataValue<int>("leaseAcquireInterval", 0);
+                    attributeTrigger.LeaseExpirationInterval = Context.GetMetadataValue<int>("leaseExpirationInterval", 0);
+                    attributeTrigger.LeaseRenewInterval = Context.GetMetadataValue<int>("leaseRenewInterval", 0);
+                    attributeTrigger.CheckpointFrequency = Context.GetMetadataValue<int>("checkpointFrequency", 0);
```

Comment:
I don't see `CheckpointDocumentCount` or whatever we end up calling it :-)

Just use this as a reminder to double-check that all attribute mappings are there.